pkg/volume/csi/csi_block.go

```diff
@@ -139,18 +139,6 @@ func (m *csiBlockMapper) SetUpDevice() (string, error) {
 	}
 	glog.V(4).Info(log("blockMapper.SetupDevice created global device map path successfully [%s]", globalMapPath))
 
-	// create block device file
```

Comment:
Delegating the creation of block device to the Driver may be a good approach.  However, before this is removed, I think it should be cleared/documented that this step will be delegated to the driver.

Reply:
I will add comments to ```SetUpDevice``` and ```MapDevice``` before calling ```NodeStageVolume``` and ```NodePublishVolume```.